src/test/mod.rs

```diff
@@ -1 +1,2 @@
+#[cfg(feature = "test-utils")]
```

Comment:
No need for that, it is already gated in `lib.rs`.

Reply:
true done